labgrid/driver/sshdriver.py

```diff
@@ -389,12 +391,28 @@ def get_status(self):
         """The SSHDriver is always connected, return 1"""
         return 1
 
+    def _ssh_prefix_append_s(self):
```

Comment:
The method name is a bit off, it does not append anything, but rather determines whether we can append `-s`. Maybe something like..

```suggestion
    def _scp_supports_explicit_sftp_mode(self):
```

Reply:
I agree. Let's go with your suggestion.